cardano-cli/src/Cardano/CLI/Orphans.hs

```diff
@@ -29,3 +33,9 @@ instance ToJSON HashableScriptData where
       [ "hash" .= hashScriptDataBytes hsd
       , "json" .= scriptDataToJsonDetailedSchema hsd
       ]
+
+-- TODO Delete me when added in API
+instance Convert Exp.Era MaryEraOnwards where
```

Comment:
This will get removed when we merge this PR: https://github.com/IntersectMBO/cardano-api/pull/701 and update CLI afterwards.